### ISSUE TITLE
Technique G73, Example 3 (SC 1.1.1)

### ISSUE THREAD

**Issue by toddmarquisboutin** [w3c/wcag#2128]
I created this issue on behalf of the [Alt Text Subgroup](https://www.w3.org/WAI/GL/task-forces/silver/wiki/Alt_Text_Subgroup) of the Silver Task Force.
### Explanation:
In [Technique G73](https://www.w3.org/TR/WCAG20-TECHS/G73.html), Example 3 reads as follows: “There is a chart. The figure caption immediately below the chart serves as a link to the long description. The Title attribute of the link makes it clear that this is a link to a long description.”

In an attempt to follow this guidance, content authors and developers might inadvertently create a subpar user experience for several reasons:

- When used as link text, the figure caption does not identify the purpose of the link.
- Mobile-browser users and keyboard-only users cannot access the Title attribute, and it is inconsistently announced by screen readers. As a result, if the Title attribute is the only text that identifies the purpose of the link, then many users will not know how to access the long description.
- If the figure caption is long, setting the entire caption as a link reduces ease of use.

### Recommendation:
Remove Example 3.

**Comment by cstrobbe** [comment]
You recommend removing example 3. However, would the following changes to the example also solve the issue?

1. In the example's title, change "Caption used as link" to "Caption contains a link".
2. In the description, replace "The figure caption (...) **serves as** a link to the long description" with ""The figure caption (...) **contains** a link to the long description."
3. In the description, replace "The Title attribute of the link makes it clear ..." with "The **link text** makes it clear ..."
